phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexUpgradeTool.java

```diff
@@ -502,6 +501,21 @@ private int rebuildIndexes(String dataTable, IndexTool indexingTool) {
         return 0;
     }
 
+    private String[] getArgValues(String schema, String baseTable, String indexName,
```

Comment:
Nit: I'll line up all parameeters - one on each line if they don't all fit in 1, instead of breaking in variable sized subsets

Reply:
I tried, it didn't look good. :(